DirectProgramming/C++SYCL_FPGA/Tutorials/GettingStarted/fpga_compile/CMakeLists.txt

```diff
@@ -19,11 +19,26 @@ set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
 ###############################################################################
 ### Customize these build variables
 ###############################################################################
-set(SOURCE_FILES src/vector_add.cpp)
+if(${PART} MATCHES 1)
+    set(SOURCE_FILES part1_cpp/src/vector_add.cpp)
+elseif(${PART} MATCHES 2)
+    set(SOURCE_FILES part2_dpcpp_functor_usm/src/vector_add.cpp)
+elseif(${PART} MATCHES 3)
+    set(SOURCE_FILES part3_dpcpp_lambda_usm/src/vector_add.cpp)
+elseif(${PART} MATCHES 4)
+    set(SOURCE_FILES part4_dpcpp_lambda_buffers/src/vector_add.cpp)
+else()
+    message(FATAL_ERROR "No matching compile parts found for part${PART}\n
+Valid parts -DPART=[ 1 - 4 ]")
+endif()
 set(TARGET_NAME vector_add)
 
 # Use cmake -DFPGA_DEVICE=<board-support-package>:<board-variant> to choose a
-# different device.
+# different device. Here are a few device examples (this list is not
```

Comment:
Please remove this change as this was removed on purpose
There should not be any mention to the PAC board in the samples anymore.